experiments/golden-results/vct-summary.txt

```diff
@@ -13,6 +13,11 @@ Calling function: Do_Aggregate_Literal
 Error message: Unhandled aggregate kind
 Nkind: N_Aggregate
 --
+Occurs: 8 times
```

Comment:
Any idea why we see some extra missing features here? Is this just that we are now processing some generics that we were previously not processing?

Reply:
@chrisr-diffblue This patch also adds rudimentary support for processing local subpackages. These weren’t processed at all before, these in particular come from a generic math package we skipped entirely before this.